### PR TITLE
[ONNX] Add GreaterOrEqual and LessOrEqual to opset 12 ONNX export

### PR DESCRIPTION
**Pull request by yaeldekel**
GreaterOrEqual and LessOrEqual were added in opset 12, this PR adds support to export these operators to ONNX instead of using "not" and "less than" or "greater than".

### CODE REVIEW

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 1]
```diff
@@ -1956,42 +1956,70 @@ def forward(self, input):
         x = torch.ones(5, 6)
         self.run_test(DimArange(), x)
 
+    def _test_compare_ops(self, model, num_inputs):
```
Comment (neginraoof) [inline test/onnx/test_pytorch_onnx_onnxruntime.py:1959]: What is the output data type in PyTorch if inputs don't have the same data type?

Comment (neginraoof) [inline test/onnx/test_pytorch_onnx_onnxruntime.py:1959]: Can you add a couple more tests for combination of input types (like float and int)?

Comment (yaeldekel) [reply test/onnx/test_pytorch_onnx_onnxruntime.py:1959]: The output type is always bool, regardless of the input types.

---
In reply to: [424530939](https://github.com/pytorch/pytorch/pull/38311#discussion_r424530939) [](ancestors = 424530939)

**Review by neginraoof — APPROVED** [review]
Thanks!

**Comment by neginraoof** [comment]
cc @houseroad for review. Thanks.

**Review by houseroad — APPROVED** [review]
Thanks

**Pull request closed without merging**